### DISCUSSION TITLE
DEV: npm start not working as expected

### DISCUSSION THREAD

**Discussion by laukik-target** [finos/git-proxy#764] [General]
I am using windows. After cloning the project.
npm start doesn't work
![image](https://github.com/user-attachments/assets/c7794f87-8044-4173-9fe0-7d814815b047)

Can someone confirm even others face same in win. I guess mac users won't face it.

If in windows others are also facing same then will raise a PR to update npm script in Package.json

**Comment by laukik-target** [comment]
Package.json -> "start": "concurrently 'npm run server' 'npm run client'",
![image](https://github.com/user-attachments/assets/8e38a7ca-5a7a-4250-8333-5b7d76950ba2)


Package.json -> "start": "concurrently \"npm run server\"' \"npm run client\"",
![image](https://github.com/user-attachments/assets/8bedef4f-a826-4c51-8538-22d1da42a803)

let me know if anyone facing the same, If yes I'll raise PR and do the needful.

**Reply by Psingle20** [reply]
yes in package.json we need to modify the start script to this `"start": "concurrently \"npm run server\" \"npm run client\""`

**Reply by JamieSlome** [reply]
@Psingle20 - feel free to open a pull request and thank you for disclosing the issue ❤️

**Comment by laukik-target** [comment]
This issue is fixed in this PR: https://github.com/finos/git-proxy/pull/763/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519
